dkeycache/App/main.cpp

```diff
@@ -57,10 +66,44 @@ void cleanup()
         delete g_la_server;
 }
 
+static void show_usage_and_exit(int code) {
+    printf("\nusage: ehsm-dkeycache -h 127.0.0.1 -p 8888\n\n");
+    exit(code);
```

Comment:
"-h" -> "-i"

"-h" for help, to print the usage

Reply:
ok,It will be change.